### PR TITLE
[INTERNATIONALIZATION] Add react-translated

### PR DESCRIPTION
**Pull request by lucianaDilizio**
- Add react-translated package.

This can be changed. I want to know your opinion. This is the easiest package for translations that I found. If you know better options let me know please 😉 

Source:
https://www.npmjs.com/package/react-translated#templated-text

### CODE REVIEW

File: src/locales/translation.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export default {
+    'Edit': {
+     	en: 'Edit',
+     	es: 'Editar'
+    },
+    'and save to reload.': {
```
Comment (fcoronelmakingsense) [inline src/locales/translation.js]: maybe is better use keys without spaces like `And_Save_To_Reload` or `and_save_to_reload`

Comment (lucianaDilizio) [reply src/locales/translation.js]: If you check the link in the source, the examples are like this. We can use it in this way if is better, of course.

File: src/App.js [hunk 2]
```diff
@@ -12,15 +13,15 @@ class App extends Component {
           <HeaderNav />
           <img src={logo} className="App-logo" alt="logo" />
           <p>
-            Edit <code>src/App.js</code> and save to reload.
+            <Translate text='Edit' /> <code>src/App.js</code> <Translate text='and save to reload.' />
```
Comment (andresmoschini) [inline src/App.js]: There should be a better way to translate text with variable content, something like:

<Translate text='Edit {content}'><code>src/App.js</code></Translate>

File: src/locales/translation.js (hunk 1 above)
Comment (andresmoschini) [reply src/locales/translation.js]: I understand your point @fcoronelmakingsense but maybe not, I would use the suggested way proposed by library owners.

**Comment by andresmoschini** [comment]
Great initiative @lucianaDilizio !!

> This can be changed. I want to know your opinion. This is the easiest package for translations that I found. If you know better options let me know please 😉

I think that it is an important think to take into account, and also how popular is it, did you research about that? Do you have some numbers to compare this library with other options? Also, in MakingSense we are already using React, did you ask what library are we using here? (maybe we would decide for someone different than they, but at least, with more information)

File: src/App.js (hunk 2 above)
Comment (lucianaDilizio) [reply src/App.js]: 🤔

Comment (andresmoschini) [reply src/App.js]: Probably it is the right way:

https://www.npmjs.com/package/react-translated#styled-text

```
<Translate text='Edit `src/App.js` and save to reload.' />
```

and if you want to make it customizable:

```
<Translate
    text='Edit `{filename}` and save to reload.'
    data={{ filename: 'src/App.js' }}
/>
```

**Comment by andresmoschini** [comment]
(I am only asking, not expressing an opinion)

Should we test it in any way?

**Comment by lucianaDilizio** [comment]
Thanks @andresmoschini! I was looking some options and this was the most simple that I found. Maybe is not too popular but it works fine. 
I will ask to my coworkers which library we are using here, I forgot that in servicies there are some projects that are using React. (Good point 😉)
I didn't create this PR to merge specifically. I just want to know your opinions and suggestions to find the best option.

**Comment by andresmoschini** [comment]
> I didn't create this PR to merge specifically. I just want to know your opinions and suggestions to find the best option.

Sure :), I am not criticizing, only asking for a little research, [npm](https://www.npmjs.com) is very useful for that. 

My concern is mainly because it could work great now but it could be not enough for us in the future, or it could even lose maintenance.

**Comment by lucianaDilizio** [comment]
Good point @andresmoschini! You're right. I will looking for something better so. Thanks!

**Comment by cbernat** [comment]
We've talked this with @lucianaDilizio and she will be looking into other alternatives before merging this.

**Comment by andresmoschini** [comment]
> We've talked this with @lucianaDilizio and she will be looking into other alternatives before merging this.

Maybe she could create another PR to compare.

**Comment by cbernat** [comment]
> > We've talked this with @lucianaDilizio and she will be looking into other alternatives before merging this.
> 
> Maybe she could create another PR to compare.

Yes, that would be great too. I've seen a lot of people use `react-intl` maybe try that one.

**Comment by andresmoschini** [comment]
Closed in favor of #13

**Pull request closed without merging**